try:
    num = 5/0
except:
    print('an error occurrd')
    raise